src/Plugin/Action/Order/ExportOrderAction.php

```diff
@@ -50,4 +56,33 @@ public function handle(array $parameters): Response
 
         return Pdk::execute(PdkActions::GET_ORDER_DATA, ['orderIds' => $orderIds]);
     }
+
+    /**
+     * @param  \MyParcelNL\Pdk\Plugin\Collection\PdkOrderCollection $orderCollection
+     *
+     * @return \MyParcelNL\Pdk\Shipment\Collection\ShipmentCollection
+     */
+    private function getShipments(PdkOrderCollection $orderCollection): ShipmentCollection
```

Comment:
Dit is een tamelijk specifieke manier van het ophalen van shipments, zou dat misschien uit de naam van de method moeten blijken?